Assets/MixedRealityToolkit.Tests/PlayModeTests/InputSystem/EnableDisableTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !WINDOWS_UWP
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests.Input
+{
+    /// <summary>
+    /// Tests that verify that various services of the MRTK support being disabled
+    /// and then re-enabled.
+    /// </summary>
+    class EnableDisableTests
+    {
+        private PointerHandler pointerHandler;
+
+        // This cube is created in test setup, and is a .2x.2.x2 sized
+        // cube located at (0, 0, 2).
+        private GameObject cube;
+
+        /// <summary>
+        /// Validates that it's possible to disable and then re-enable the input system,
+        /// and that the gaze subsystem is inactive when the input system is disabled
+        /// and working again once the input system is re-enabled.
+        /// </summary>
+        [UnityTest]
+        public IEnumerator TestGazeInputSystemEnableDisable()
+        {
+            // Assert that the initial state of the cube is one that hasn't had any relevant
+            // focus events
+            Assert.AreEqual(0, pointerHandler.state.numFocusEnter);
+
+            // The input system is enabled by default, so we first disable it.
+            MixedRealityServiceRegistry.DisableService<IMixedRealityInputSystem>();
+            yield return null;
+
+            CameraCache.Main.transform.LookAt(cube.transform);
+            yield return null;
+
+            // At this point, looking at the cube would normally have triggered 
+            // a numFocusEnter increment, but because the input system is disabled
+            // this shouldn't do anything. In addition, the GazeProvider should still
+            // be valid (i.e. non-null) but its target shouldn't have updated.
+            Assert.AreEqual(0, pointerHandler.state.numFocusEnter);
+            Assert.IsNotNull(CoreServices.InputSystem.GazeProvider);
```

Comment:
I would expect that even with input system disabled, that gaze would be running. In my mind, gaze is a sensor stream that is combined with input in MRTK, but should not be.